agent/bench-scripts/pbench-linpack

```diff
@@ -201,25 +209,56 @@ for thread in ${threads//,/ }; do
 	mkdir -p "${iteration_dir}"
 	record_iteration ${count} ${thread} ${iteration}
 	echo "Starting iteration ${iteration}"
-
-	# Pre-processing: generate the linpack.input dirs (in /var/lib/pbench-agent/tmp)
+	# Pre-processing: generate the linpack.input files
 	preprocess-iteration "${iteration_dir}" "${thread}"
+	# Distribute the linpack.input files to remote clients if necessary
```

Comment:
If you need to make a change for some other reason, I suggest replacing `linpack.input` with `linpack input` (replace the dot with a space).

Reply:
Fixed.

Reply:
You got one of them...you didn't want to fix the other?  🤪

Reply:
Fixed.